src/httpd.c

```diff
@@ -908,6 +912,9 @@ httpd_request_parse(struct evhttp_request *req, struct httpd_uri_parsed *uri_par
   // Find a handler for the path
   for (i = 0; uri_map[i].handler; i++)
     {
+      if (uri_map[i].method > 0 && req_method > 0 && !(req_method & uri_map[i].method))
```

Comment:
'if (uri_map[i].method && req_method && ...' might be easier to read. You could also consider adding a comment.